package.json

```diff
@@ -44,9 +44,12 @@
     "renameschema": "scripts/rename-schema.sh",
     "presite": "npm run data && npm run build:site && npm run build:toc && npm run build:versions && scripts/create-example-pages",
     "site": "bundle exec jekyll serve --incremental",
-    "prettierbase": "prettier '{src,test,test-runtime,site,typings}/**/*.{ts,md,css}'",
-    "format": "tslint -p . --fix && npm run prettierbase -- --write",
-    "lint": "tslint -p . && npm run prettierbase -- --check",
+    "tsc:src": "tsc -b src/tsconfig.src.json",
```

Comment:
I'm confused about these additions to package.json scripts? They don't seem related to this PR?

Reply:
I missed some changes while I was rebasing from the latest version of master and resolving merge conflicts. These changes just undos those changes such that `package.json` as a whole has no diff in this PR.